otelcol/command_components.go

```diff
@@ -31,19 +36,60 @@ func newComponentsCommand(set CollectorSettings) *cobra.Command {
 
 			components := componentsOutput{}
 			for con := range set.Factories.Connectors {
-				components.Connectors = append(components.Connectors, con)
+				components.Connectors = append(components.Connectors, componentWithStability{
+					Name: con,
+					StabilityLevels: map[string]string{
+						"logstologs":    set.Factories.Connectors[con].LogsToLogsStability().String(),
```

Comment:
should the keys be more user friendly? ie. `logs to logs` or `logs-to-logs`?

Reply:
Done!